Documentation/Contributors/BuildGuide/README.md

```diff
@@ -22,13 +23,15 @@
    * Yes
       * Clone the cesium repo, e.g., `git clone git@github.com:AnalyticalGraphicsInc/cesium.git`
       * Make changes in a branch, e.g., `git checkout -b my-feature`.
+* For more about our workflow, see [github pull request workflows](http://cesiumjs.org/2013/10/08/GitHub-Pull-Request-Workflows/).
+
 
 ## Build the Code
 
 Prerequisites:
  * Install [Node.js](http://nodejs.org/) on your system.  Building Cesium requires Node 4.x or newer.
 
-Cesium uses npm modules for development, so after syncing, you need to run `npm install` from the Cesium root directory:
+Cesium uses [npm modules](https://docs.npmjs.com/getting-started/creating-node-modules) for development, so after syncing, you need to run `npm install` from the Cesium root directory:
```

Comment:
I would link this to https://docs.npmjs.com/getting-started/what-is-npm instead.  I think that's more useful for people that are new to npm